lib/screens/vehicles/parser.ex

```diff
@@ -77,4 +77,13 @@ defmodule Screens.Vehicles.Parser do
   defp parse_occupancy_status("NO_DATA_AVAILABLE"), do: :no_data_available
   defp parse_occupancy_status("NOT_ACCEPTING_PASSENGERS"), do: :not_accepting_passengers
   defp parse_occupancy_status(_), do: nil
+
+  defp parse_carriage_occupancy_status("NO_DATA_AVAILABLE"), do: :no_data
+  defp parse_carriage_occupancy_status("MANY_SEATS_AVAILABLE"), do: :not_crowded
+  defp parse_carriage_occupancy_status("FEW_SEATS_AVAILABLE"), do: :not_crowded
+  defp parse_carriage_occupancy_status("STANDING_ROOM_ONLY"), do: :some_crowding
+  defp parse_carriage_occupancy_status("CRUSHED_STANDING_ROOM_ONLY"), do: :crowded
+  defp parse_carriage_occupancy_status("FULL"), do: :crowded
+  defp parse_carriage_occupancy_status("NOT_ACCEPTING_PASSENGERS"), do: :closed
+  defp parse_carriage_occupancy_status(_), do: nil
```

Comment:
This may have been too impulsive of a change, cause you're right. This does feel weird. Would it be better to add this conversion as a util function instead?

Reply:
A util function could be good, yeah! Reusable but also doesn't force this mapping from v3 API classification to our classification in all cases.